src/Styleguide/Pages/__stories__/Pages.story.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { storiesOf } from "storybook/storiesOf"
+import { storiesOf } from "@storybook/react"
```

Comment:
Swapped this with normal `storiesOf` since `/Pages` will handle their own providers.